matrix_sdk/Cargo.toml

```diff
@@ -15,9 +15,10 @@ features = ["docs"]
 rustdoc-args = ["--cfg", "feature=\"docs\""]
 
 [features]
-default = ["encryption", "sled_cryptostore", "sled_state_store", "require_auth_for_profile_requests", "native-tls"]
+default = ["encryption", "qrcode", "sled_cryptostore", "sled_state_store", "require_auth_for_profile_requests", "native-tls"]
```

Comment:
Please document this feature flag in [lib.rs](https://github.com/matrix-org/matrix-rust-sdk/blob/master/matrix_sdk/src/lib.rs#L37).

Reply:
Actually the feature flags got just moved to the readme.